src/pkg/errors.go

```diff
@@ -14,3 +14,9 @@ func (e ErrDeploymentFailed) Error() string {
 	}
 	return fmt.Sprintf("deployment failed%s: %s", service, e.Message)
 }
+
+type ErrDeploymentCompleted struct{}
```

Comment:
This is not an error, so I rather we leave this private to the code where we need it.